docs/content/dependencies.md

```diff
@@ -112,8 +110,8 @@ how it looks:
 
 To fix this, we'll need to specify `depends` on our `file.content`. `depends` is
 a list of resources in the current module that must be successfully walked
-before walking ours. They're specified as the resource type, a dot, then the
-resource name. So `task "names"` above becomes `task.names`.
+before walking the containing resource. They're specified as the resource type,
```

Comment:
Why don't we remove references to walking here. I could see them confusing new users. "… that must be successful before beginning the current resource" could work?